src/core/transport/transport.h

```diff
@@ -72,6 +72,12 @@ typedef struct grpc_transport_stream_op {
 
   grpc_stream_op_buffer *recv_ops;
   grpc_stream_state *recv_state;
+  /** The number of bytes this peer is currently prepared to receive.
+
+      Bytes offered are used to replenish per-stream flow control windows.
+      Offers are not retractable: if 5 bytes are offered and no bytes are read,
+        a later offer of 3 bytes still implies that 5 have been offered. */
```

Comment:
This comment seems to address an implied API to update the max_recv_bytes rather than the semantics of this state variable. Here is my proposed rephrasing:

The number of bytes this peer is currently prepared to receive.
These bytes will be eventually used to replenish per-stream flow control windows. 


Alternately, you can reuse the corresponding comment from src/core/transport/chttp2/internal.h